chat-components/src/main/java/com/pubnub/components/chat/service/channel/DefaultChannelService.kt

```diff
@@ -2,23 +2,41 @@ package com.pubnub.components.chat.service.channel
 
 import com.pubnub.api.PubNub
 import com.pubnub.components.data.channel.DBChannel
+import com.pubnub.framework.data.ChannelGroupId
+import com.pubnub.framework.data.ChannelId
 import com.pubnub.framework.service.error.Logger
 
 class DefaultChannelService(
     private val pubNub: PubNub,
     private val logger: Logger,
 ) : ChannelService<DBChannel> {
 
-    private var subscribedChannels: List<String> = listOf()
+    private var subscribedChannels: List<ChannelId> = listOf()
+    private var subscribedChannelGroups: List<ChannelGroupId> = listOf()
 
-    override fun bind(vararg channels: String) {
-        logger.i("Subscribing to the channels '${channels.joinToString()}'")
-        subscribedChannels = listOf(*channels)
-        pubNub.subscribe(channels = subscribedChannels, withPresence = true)
+    /**
+     * Subscribes to channels and channel groups
+     *
+     * @param channels - Channels to subscribe/unsubscribe. Either channel or channelGroups are required.
+     * @param channelGroups - Channel groups to subscribe/unsubscribe. Either channelGroups or channels are required.
+     * @param withPresence - Also subscribe to related presence channel.
+     */
+    override fun bind(channels: List<ChannelId>, channelGroups: List<ChannelGroupId>, withPresence: Boolean) {
```

Comment:
@kazydek Default service updated

Reply:
No update is needed (we don't have that info in the docs)